keycloak/sms-provider/src/main/java/org/sunbird/sms/msg91/Msg91SmsProvider.java

```diff
@@ -0,0 +1,318 @@
+package org.sunbird.sms.msg91;
+
+import com.amazonaws.util.StringUtils;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpHost;
+import org.apache.http.StatusLine;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.ByteArrayEntity;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
```

Comment:
Do we really need all these imports from http ?

Reply:
@coolbung Yes. IDE will automatically import only the ones needed. So these are needed ones.